BACnetServerExample/BACnetServerExample.cpp

```diff
@@ -2075,6 +2129,16 @@ bool CallbackDeviceCommunicationControl(const uint32_t deviceInstance, const uin
 	return true;
 }
 
+void CallbackLogDebugMessage(const char* message, const uint16_t messageLength, const uint8_t messageType) {
+	// This callback is called when the CAS BACnet Stack logs an error or info message
```

Comment:
Just dump the string to stdout in the call-back function. No need to make a global function